index.html

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
The reason I'm using HTML is because I like the idea of having two things beside each other (and I have no idea how to do that in Markdown). Don't know if it's possible in markdown. 

I also used templating/etc. by Foundation Zurb (simply add the css file, and special classes for the markup), and it automatically makes it look nice on each screen size (from mobile to 4K and everything in between). Not sure if this is compatible with markdown / if markdown does the same?